### DISCUSSION TITLE
WWFF: Autofill locator and QTH field

### DISCUSSION THREAD

**Discussion by m0urs** [magicbug/Cloudlog#1701] [Ideas / Feature Requests]
I really like the separate WWFF field which has been introduced some time ago. It would be even better, if, after entering a WWFF reference, the locator and QTH field would be automatically filled, if not already populated. Currently I do these steps every time manually and it would be great help, if this could be done automatically.

Just like it has  already been implemented with the SOTA reference.

All the required information would be available as a CSV file from https://www.cqgma.org/wwff.csv

I already looked a bit into the code but that is, unfortunately, too difficult for me to do by myself :-(

**Comment by magicbug** [comment]
If the data is available i don't see why not, think this fine has more data than the file we download at the moment so might take some reworking.

**Comment by magicbug** [comment]
Made this into an issue https://github.com/magicbug/Cloudlog/issues/1702
